### PR TITLE
Fix mangling CSS values that use `!important`

### PR DESCRIPTION
**Pull request by ericcornelissen**

### CODE REVIEW

File: packages/core/src/languages/css/__tests__/css-values.test.ts [hunk 1]
```diff
@@ -64,6 +64,27 @@ suite("CSS - CSS Value Expression Factory", function() {
         },
       ],
     },
+    {
+      name: "edge cases",
```
Comment (ericcornelissen) [inline packages/core/src/languages/css/__tests__/css-values.test.ts:68]: Rename this to `"with !important"`

```suggestion
      name: "with !important",
```

File: packages/core/src/languages/html/__tests__/css-values.test.ts [hunk 2]
```diff
@@ -64,6 +64,27 @@ suite("HTML - CSS Value Expression Factory", function() {
         },
       ],
     },
+    {
+      name: "edge cases",
```
Comment (ericcornelissen) [inline packages/core/src/languages/html/__tests__/css-values.test.ts:68]: Rename this to `"with !important"`

```suggestion
      name: "with !important",
```
